examples_tests/06.MeshLoaders/main.cpp

```diff
@@ -304,7 +345,10 @@ class MeshLoadersApp : public ApplicationBase
         auto& fence = frameComplete[resourceIx];
 
         if (fence)
+        {
             while (logicalDevice->waitForFences(1u, &fence.get(), false, MAX_TIMEOUT) == video::IGPUFence::ES_TIMEOUT) {}
```

Comment:
block for fences.

Reply:
No? `blockForFences` always return `true` and judging from the `assert` in `blockForFences` when it returns `false`, it seems, it is not expected to return `false`? Then how would you expect the loop to ever exit?

Reply:
obviously remove the whole `while(logicalDevice->waitForFences...` and replace with `logicalDevice->blockForFences`

the loop has to go

Reply:
Yeah right ofcourse 🤦‍♂️